packages/models-library/src/models_library/api_schemas_directorv2/dynamic_services_service.py

```diff
@@ -39,19 +39,7 @@ class ServiceDetails(CommonServiceDetails):
         description="predefined path where the dynamic service should be served. If empty, the service shall use the root endpoint.",
         alias="service_basepath",
     )
-
-    class Config:
-        allow_population_by_field_name = True
-        schema_extra: ClassVar[dict[str, Any]] = {
-            "example": {
-                "key": "simcore/services/dynamic/3dviewer",
-                "version": "2.4.5",
-                "user_id": 234,
-                "project_id": "dd1d04d9-d704-4f7e-8f0f-1ca60cc771fe",
-                "node_uuid": "75c7f3f4-18f9-4678-8610-54a2ade78eaa",
-                "basepath": "/x/75c7f3f4-18f9-4678-8610-54a2ade78eaa",
-            }
-        }
+    model_config = ConfigDict(populate_by_name=True)
```

Comment:
Q: I see you are removing schema_extra. I believe we use it in many places, such as in tests, to retrieve examples of this Pydantic model. Is there any way to keep it?

Reply:
See: https://github.com/ITISFoundation/osparc-simcore/pull/6333#discussion_r1751396302